python/test/unit/io/test_vtk.py

```diff
@@ -10,24 +10,27 @@
 
 from dolfin import (MPI, Function, FunctionSpace, MeshFunction,
                     TensorFunctionSpace, UnitCubeMesh, UnitIntervalMesh,
-                    UnitSquareMesh, VectorFunctionSpace, VTKFile)
-from dolfin_utils.test import fixture, skip_in_parallel, tempdir
+                    UnitSquareMesh, VectorFunctionSpace)
+from dolfin_utils.test.skips import skip_in_parallel
+from dolfin_utils.test.fixtures import tempdir
+
+from dolfin.io import VTKFile
 
 assert (tempdir)
 
 
 # VTK file options
-@fixture
+@pytest.fixture
 def file_options():
     return ["ascii", "base64", "compressed"]
```

Comment:
I thought compressed and base64 had been removed?

Reply:
Right now we do not use `file_options` in the tests, as the initializer in CPP do not support filetype. I just updated the syntax from the previous dolfin in case we decide to re-add this. You can see that there are a lot of tests in this file that are marked with xfail.